README.md

```diff
@@ -237,6 +238,7 @@ Here are all the samples which are deployed on Microsoft Azure and you can try i
 [appcheckinlocation#csharp]:samples/app-checkin-location/csharp
 [checkinlocation#nodejs]:samples/app-checkin-location/nodejs
 [productinspection#nodejs]:samples/tab-product-inspection/nodejs
+[memessagereminder#nodejs]:samples/bot-message-reminder/nodejs
```

Comment:
Please renmae the folde to msgext-message-reminder. Please do the same for node project.